src/EventStore.Core/Services/Storage/StorageWriterService.cs

```diff
@@ -92,17 +94,20 @@ public class StorageWriterService<TStreamId> : IHandle<SystemMessage.SystemInit>
 			IIndexWriter<TStreamId> indexWriter,
 			IRecordFactory<TStreamId> recordFactory,
 			INameIndex<TStreamId> streamNameIndex,
+			INameIndex<TStreamId> eventTypeIndex,
 			ISystemStreamLookup<TStreamId> systemStreams,
 			IEpochManager epochManager,
 			QueueStatsManager queueStatsManager,
-			IPartitionManager partitionManager) {
+			IPartitionManager partitionManager,
+			TStreamId emptyEventTypeId) {
```

Comment:
might be nice to group this argument next to the eventTypeIndex arg (same in ClusterStorageWriterService)